utils/collect_os_info.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+Description="The purpose of this script is to collect system information for bug reports.\n
+Submit issues to https://github.com/osm-search/Nominatim/issues"
+
+
+####### Gather the Information ##################################################
+# Separate the information gathering from the report generation.  Dividing these
+# makes it easier to make trivial changes by not have to learn the other portion
+# of this script.
+
+# Nominatium version
+# NOTE: Getting this version will NOT work if it is being ran from in another
+# folder than Nominatim/utils.  It call python3 to import version.py locally and
+# prints it in the version format. 
+NominatimVersion=`cd ../nominatim/ && python3 -c "import version; print('{0[0]}.{0[1]}.{0[2]}-{0[3]}'.format(version.NOMINATIM_VERSION))"`
+
+# PostgreSQL version
+PostgreSQLVersion=`postgres --version`
+if [ "$?" -ne "0" ]
+then
+  PostgreSQLVersion="Not installed"
+fi
+
+# - PostGIS version:
+# The command for this should look something like this:
+#       psql -U nominatim -d mydatabase -c 'SELECT PostGIS_full_version();'
+# ASSUME the username is nominatim
+# This needs to be ran under the account with the appropriate permissions.
+# This has been left blank.
+PostGISVersion=
```

Comment:
Assuming that the default postgis3 version is installed, maybe something like this:

```sh
grep '^default_version' "$(pg_config --sharedir)/extension/postgis-3.control"
```